### PR TITLE
vsock: Correct parent bus issue

### PR DESCRIPTION
**Pull request by vivianQizhu**
1.  qemu_vm: add parent_bus for vsock pci device
    It will not have bus automatically, so no address being issued,
     therefor will hit error when add extra pcie-root-port for pcie bus

2.  qdevices: add missed comma in pcie_devices list
    Miss a ',' after "vhost-vsock-pci", which will lead all components
    after it be treated as pcie device, be plugged into pcie.0 directly

id: 1694901
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
Buggy cmd line:
```
-device vhost-vsock-pci,id=vhost_vsock0,guest-cid=3 \

-device virtio-scsi-pci,id=virtio_scsi_pci0,bus=pcie.0,addr=0x3 \
```
Now:
```
-device pcie-root-port,id=pcie.0-root-port-3,slot=3,chassis=3,addr=0x3,bus=pcie.0 \
-device virtio-scsi-pci,id=virtio_scsi_pci0,bus=pcie.0-root-port-3,addr=0x0 \

-device pcie-root-port,id=pcie.0-root-port-5,slot=5,chassis=5,addr=0x5,bus=pcie.0 \
-device vhost-vsock-pci,id=vhost_vsock0,guest-cid=3,bus=pcie.0-root-port-5,addr=0x0 \
```

 (1/1) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.0.0.x86_64.io-github-autotest-qemu.vsock_test.q35: PASS (55.16 s)

**Comment by vivianQizhu** [comment]
@yanan-fu @xiangchunfu @luckyh Please help review, thanks.

File: virttest/qemu_devices/qdevices.py [hunk 1]
```diff
@@ -1839,7 +1839,7 @@ def is_direct_plug(self, device):
         :param device: the QBaseDevice object
         :return: bool value for directly plug or not.
         """
-        pcie_devices = ["virtio-net-pci", "virtio-blk-pci", "vhost-vsock-pci"
+        pcie_devices = ["virtio-net-pci", "virtio-blk-pci", "vhost-vsock-pci",
```
Comment (luckyh) [inline virttest/qemu_devices/qdevices.py]: Good catch.

**Comment by yanan-fu** [comment]
Acked-by: Yanan Fu <yfu@redhat.com>

**Comment by xiangchunfu** [comment]
repeat1.Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.x86_64.io-github-autotest-qemu.vsock_test.q35

And I checked debug log.  It works well. 
Acked-by:Xiangchun Fu xfu@redhat.com

**Comment by vivianQizhu** [comment]
Thanks all.

**Pull request merged**